cardano-api/src/Cardano/Api/ScriptData.hs

```diff
@@ -362,11 +361,6 @@ scriptDataToJsonNoSchema = conv
     conv :: ScriptData -> Aeson.Value
     conv (ScriptDataNumber n) = Aeson.Number (fromInteger n)
     conv (ScriptDataBytes bs)
-      | Right s <- Text.decodeUtf8' bs
```

Comment:
Why? This is plutus script data not tx metadata.

Reply:
You sent me tx-metadata docs before. I supposed, tx-metadata rules apply here. Don't they?